src/Clerk.Webhooks/Clerk.Webhooks.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.2.2" />
+      <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0" />
+      <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+    </ItemGroup>
+
```

Comment:
We need to add below. Version is for configuring the version to never be default (1.0.0). The versionSuffix we might need to do somehting. But since dotnet pack -p:Version not can take the suffix in as version, this would be the way to ensure that alpha is in all current release.

`<PropertyGroup>
    <Version></Version>
    <VersionSuffix>alpha</VersionSuffix>
</PropertyGroup>`